article/app/views/package.scala

```diff
@@ -10,10 +10,16 @@ import views.support.cleaner.{AmpEmbedCleaner, VideoEmbedCleaner, CmpParamCleane
 object MainMediaWidths {
 
   def apply(article: Article): layout.WidthsByBreakpoint = {
-    (article.hasShowcaseMainElement, article.isFeature) match {
-      case (true, true) => MainMedia.FeatureShowcase
-      case (true, _) => MainMedia.Showcase
-      case _ => MainMedia.Inline
+    if (article.hasShowcaseMainElement) {
+      if (article.isFeature) {
+        MainMedia.FeatureShowcase
+      } else {
+        MainMedia.Showcase
+      }
+    } else if (article.contentType == "LiveBlog") {
```

Comment:
@jennysivapalan How would you write this? I struggled because I needed to match three things: `article.hasShowcaseMainElement`, `article.isFeature`, and `article.contentType == "LiveBlog"`.